Mathlib/Tactic/ToExpr.lean

```diff
@@ -37,6 +37,11 @@ attribute [-instance] Lean.instToExprProd
 
 deriving instance ToExpr for Prod
 
+open System in
+instance : ToExpr FilePath where
+  toTypeExpr := mkConst ``FilePath
+  toExpr path := mkApp (mkConst ``FilePath.mk) (toExpr path.1)
```

Comment:
Would `deriving instance ToExpr for FilePath` have worked here?

Reply:
Took me a moment to discover that `whatsnew in` was the easiest way to see what `deriving instance` does here, but yes, it is correct. I've changed it out.